src/main/asciidoc/jpa.adoc

```diff
@@ -521,20 +521,42 @@ repo.findByAndSort("stark", Sort.by("LENGTH(firstname)"));            <2>
 repo.findByAndSort("targaryen", JpaSort.unsafe("LENGTH(firstname)")); <3>
 repo.findByAsArrayAndSort("bolton", Sort.by("fn_len"));               <4>
 ----
+
 <1> Valid `Sort` expression pointing to property in domain model.
-<2> Invalid `Sort` containing function call. Throws Exception.
+<2> Invalid `Sort` containing function call.
+Throws Exception.
 <3> Valid `Sort` containing explicitly _unsafe_ `Order`.
 <4> Valid `Sort` expression pointing to aliased function.
 ====
 
+[[jpa.query-methods.scroll]]
+=== Scrolling large Query Results
+
+When working with large data sets, <<repositories.scrolling,scrolling>> can help to process those results efficiently without loading all results into memory.
```

Comment:
I extracted several impulses into spring-projects/spring-data-commons#2804 so that we can include less store-specific details here and can link to the actual elaboration on our scrolling methods.